scripts/auto-rebase/rebase.sh

```diff
@@ -159,6 +156,8 @@ write_lvms_images_for_arch(){
 }
 
 update_lvms_images(){
+    title "Updating LVMS manifests"
```

Comment:
```suggestion
    title "Updating LVMS images"
```

Reply:
Good catch, I'll fix that in the next revision.